webui/react/src/pages/ExperimentDetails/ExperimentActions.tsx

```diff
@@ -53,7 +53,7 @@ const ExperimentActions: React.FC<Props> = ({ experiment, onClick, onSettled }:
   });
 
   const handleArchive = useCallback((archive: boolean) => async (): Promise<void> => {
-    setButtonStates(state => ({ ...state, archive }));
+    setButtonStates(state => ({ ...state, archive: true }));
```

Comment:
**block:** This is probably something I did, but `archive` and all the other properties of `setButtonStates` should be uppercase, or we can just change all the initial values to be lowercase (in lines 46-52)`.